cartocolor.js

```diff
@@ -1097,6 +1097,51 @@ var cartocolor = {
             "diverging"
         ]
     },
+    "TealRose": {
```

Comment:
well, actually, `Temps` is the old `TealRose` and `TealRose` I modified to blend better.